Makefile

```diff
@@ -172,19 +187,36 @@ $(shell mkdir -p build/baserom build/assets/text $(foreach dir,$(SRC_DIRS) $(UND
 build/src/libultra/libc/ll.o: OPTFLAGS := -Ofast
 build/src/%.o: CC := $(CC) -fexec-charset=euc-jp
 
+build/src/overlays/actors/ovl_Item_Shield/%.o: OPTFLAGS := -O2
+build/src/overlays/actors/ovl_En_Part/%.o: OPTFLAGS := -O2
+build/src/overlays/actors/ovl_Item_B_Heart/%.o: OPTFLAGS := -O0
+build/src/overlays/actors/ovl_Bg_Mori_Hineri/%.o: OPTFLAGS := -O0
+
 #### Main Targets ###
 
 all: $(ROM)
 
+compress: $(ROMC)
+
+wad:
+	$(MAKE) compress CFLAGS="-DCONSOLE_WIIVC $(CFLAGS) -fno-reorder-blocks -fno-optimize-sibling-calls" CPPFLAGS="-DCONSOLE_WIIVC $(CPPFLAGS)"
+	@echo 45e | tools/gzinject/gzinject -a genkey -k common-key.bin >/dev/null
```

Comment:
what does ``>/dev/null`` do exactly? I was wondering that the last time and forgot to ask

Reply:
i think that was something spaceworld 97 did

Reply:
welp, if it works it work I guess lol